lib/plugins/amplify-flutter-predictions/amplify_predictions.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:toktik/plugins/amplify-flutter-predictions/method_channel_amplify_predictions.dart';
+
+//  * will need to add these line into amplifyconfiguration.dart to make it work
+//  * "predictions": {
+//         "plugins": {
+//             "awsPredictionsPlugin": {
+//                 "defaultRegion": "us-west-2",
+//                 "convert": {
+//                     "transcription": {
+//                         "language": "en-US",
+//                         "region": "us-west-2",
+//                         "defaultNetworkPolicy": "auto"
+//                     }
+//                 }
+//             }
+//         }
+//     }
+class AmplifyPredictions {
```

Comment:
Thanks for introducing this! If it is painful to integrate prediction directly, we could have it as mobile->lambda->prediction. So that AI functionality will be accessed as a common microservice. Also, I suppose the server side needs to download the podcast. How is this handled now?